src/utils/websockets.js

```diff
@@ -38,11 +40,21 @@ export function connectWS(topic, onMessageCallback) {
   };
 
   const connect = () => {
+    let reconnectCounter = 0;
     this.sockClient = new Client({
       brokerURL: config.WS_URL,
-      // debug: function(str) {
-      //   console.log('debug: ', str);
-      // },
+      debug(str) {
+        // console.log('debug: ', str);
+        // -- detect reconnect and increment the reconnect counter
+        if (str.includes('reconnect')) {
```

Comment:
isn't there any much more reliable way of detecting a  reconnect attempt. String search are usually very fragile.

Reply:
I'm not aware of any other. I hardly found this one